feature/bgp/policybase/otg_tests/link_bandwidth_test/README.md

```diff
@@ -27,7 +27,8 @@ communities to routes based on a prefix match.
       * prefix-set-1 with 2 ipv4 and 2 ipv6 routes without communities.
       * prefix-set-2 with 2 ipv4 and 2 ipv6 routes with communities `[ "100:100" ]`.
       * prefix-set-3 with 2 ipv4 and 2 ipv6 routes with extended communities `[ "link-bandwidth:100:0" ]`.
-
+  * Configure Send community knob to IBGP neigbour to advertise the communities to IBGP peer 
+    * use `/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/config/send-community`
```

Comment:
Also add this path to the the path list at the bottom:

https://github.com/openconfig/featureprofiles/pull/2796/files#diff-c3bfc25b0cc3b7bad33c65e142db4016feb5d91ef8c597986f6f390780c9b1b7R164

Reply:
As suggested updated path at the bottom

Thanks